### ISSUE TITLE
GLTF export support

### ISSUE THREAD

**Issue by unit-a-user** [erich666/Mineways#95]
I really need gltf export support. Mineways doesn't have a single normal format, and there isn't a single normal way to convert to gltf. Generally! So please add support directly.

**Comment by JMY1000** [comment]
You should be able to import the OBJ file into Blender and export it as a GLTF file.

**Comment by unit-a-user** [comment]
Yeah, but obj not storing correct roughness info. For example, glass are glossy reflective.

**Comment by unit-a-user** [comment]
I do not want to give links to an external site, this is generally unacceptable anywhere, and bots are banned everywhere for this, but there are specific PBR extensions for OBJ.
http://exocortex.com/blog/extending_wavefront_mtl_to_support_pbr

**Comment by unit-a-user** [comment]
Test

**Comment by erich666** [comment]
Sorry to just notice this now (Github, why are you not sending me emails?). I've been tempted to add glTf support, but to be honest I'd rather spend my (limited) time supporting USD, which I think is the future. That said, I might bite the bullet and add glTf, but we'll see. Reminder to self: 
https://github.com/jessey-git/fx-gltf or https://github.com/syoyo/tinygltf or https://github.com/jkuhlmann/cgltf

**Comment by erich666** [comment]
Really, my advice is to use USD export, in the long-term. USD has a similar PBR material model in its UsdPreviewSurface, e.g., see this test site: https://asset-explorer.needle.tools/

Blender 4.0.0, for example, will have much better USD support when it comes out.

That said, I'll think about adding glTF export - it'd be educational. But, a lot of work, testing to see if I'm exporting things well.

I'm asking around for the best free USD to glTF exporter out there currently. I'll post it here if I find anything good.
